### PR TITLE
New spanish translations

### PR DESCRIPTION
**Pull request by jdsajdas**
-hitResultNoEffect
-hitResultCriticalHit
-hitResultSuperEffective
-hitResultNotVeryEffective
-hitResultOneHitKO

-accuracy

translated

### CODE REVIEW

File: src/locales/es/battle.ts [hunk 1]
```diff
@@ -14,10 +14,10 @@ export const battle: SimpleTranslationEntries = {
...
   "sendOutPokemon": "¡Adelante, {{pokemonName}}!",
-  "hitResultCriticalHit": "A critical hit!",
-  "hitResultSuperEffective": "It's super effective!",
-  "hitResultNotVeryEffective": "It's not very effective…",
-  "hitResultNoEffect": "It doesn't affect {{pokemonName}}!",
+  "hitResultCriticalHit": "!Un golpe crítico!",
+  "hitResultSuperEffective": "!Es muy eficaz!",
+  "hitResultNotVeryEffective": "No es muy eficaz…",
+  "hitResultNoEffect": "No afecta a {{pokemonName}}!",
   "hitResultOneHitKO": "It's a one-hit KO!",
```
Comment (TempsRay) [inline src/locales/es/battle.ts:21]: Does this one also need to be translated?

Comment (jdsajdas) [reply src/locales/es/battle.ts:21]: Why not?

Comment (TempsRay) [reply src/locales/es/battle.ts:21]: I meant line 21 "It's a one-hit KO!" should also be in Spanish

Comment (jdsajdas) [reply src/locales/es/battle.ts]: You are right

**Review by TempsRay — APPROVED** [review]
The translations look fine to me but I don't know if they're 100% what the in game text is. I could only find Spanish for the critical hit message.

**Comment by jdsajdas** [comment]
Are screenshots useful?

**Comment by TempsRay** [comment]
If you have screenshots or photos of the actual spanish text in the most recent Pokemon games that would be best

**Comment by jdsajdas** [comment]
https://youtu.be/QmZ6iHvzuQQ?si=SyyLy5k8NV9cHtek
Time 5:50, KO

IMG-20240514-WA0141
Critic hit

IMG-20240514-WA0142
Not very effective

IMG-20240514-WA0143
No efect

IMG-20240514-WA0144
Super effective

IMG-20240514-WA0145
Accuracy

**Review by TempsRay — APPROVED** [review]
LGTM!

**Review by bennybroseph — APPROVED** [review]

**Pull request merged**